tests/queryset_pickle/tests.py

```diff
@@ -115,6 +115,38 @@ def test_pickle_prefetch_related_idempotence(self):
         groups = pickle.loads(pickle.dumps(groups))
         self.assertSequenceEqual(groups, [g])
 
+    def test_pickle_prefetch_queryset_not_evaluated(self):
```

Comment:
My philosophy is that it's easier to see what the commit is fixing if tests that already work aren't part of the same commit. Also, if a commit has to be reverted, the additional test coverage won't be removed by default.